src/callstack.cr

```diff
@@ -17,18 +17,9 @@ require "callstack/lib_unwind"
   require "debug/dwarf"
 {% end %}
 
-# `caller` returns the current execution stack (a backtrace) in the form
-# of an Array of strings:
+# Returns the current execution stack (a backtrace) as `Array(String)`.
```

Comment:
As per @straight-shoota suggestion, I think we should mention that it's the execution stack of the current fiber.